Jenkinsfile

```diff
@@ -19,6 +25,13 @@ elifePipeline {
             stage 'Merge to master', {
                 elifeGitMoveToBranch commit, 'master'
             }
+
+            stage 'Push unstable image', {
+                def image = DockerImage.elifesciences(this, 'sciencebeam-trainer-grobid', fullImageTag)
+                def unstable_image = image.addSuffixAndTag('_unstable', fullImageTag)
+                unstable_image.tag("${grobidTag}-latest").push()
```

Comment:
`1.2.3-latest` seems special but it isn't. Probably `latest` could be omitted. Examples from `python`:
```
3.7.3-alpine3.9
3.7-alpine3.9
3-alpine3.9
alpine3.9
3.7.3-alpine
3.7-alpine
3-alpine
alpine
```
so this could be `sciencebeam-trainer-grobid_unstable:0.4.4-12345678` (assuming `12345678` is the commit`) plus `sciencebeam-trainer-grobid_unstable:0.4.4` which is the `latest` (default) with respect to commits to this repository.